lib/tlscontext.c

```diff
@@ -846,6 +852,43 @@ tls_context_setup_sigalgs(TLSContext *self)
   return TRUE;
 }
 
+static gboolean
+_apply_cmd_context(TLSContext *self)
+{
+  gboolean result = TRUE;
+
+  for (GList *next_pair = self->conf_cmds_list; next_pair; next_pair = next_pair->next)
+    {
+      gchar *cmd = next_pair->data;
+      next_pair = next_pair->next;
+      gchar *value = next_pair->data;
+
+      int add_result = SSL_CONF_cmd(self->ssl_conf_ctx, cmd, value);
+      if (add_result < 1)
+        msg_error("Error setting up SSL conf context, invalid cmd or value in config list",
+                  evt_tag_str("cmd", cmd),
+                  evt_tag_str("value", value),
+                  tls_context_format_location_tag(self));
+      result = result && (add_result >= 1);
+    }
```

Comment:
It is undocumented, but we also need a `SSL_CONF_CTX_finish()` call to fill CA lists.